src/reanimated2/animation/repeat.ts

```diff
@@ -39,6 +40,13 @@ export const withRepeat = function <T extends AnimationObject>(
 ): Animation<RepeatAnimation> {
   'worklet';
 
+  const a =
```

Comment:
We already define this inside defineAnimation.
```suggestion
  const nextAnimation =
```
Please name this variable `nextAnimation` instead of `a` and remove the second declaration in line 55.

Reply:
This cannot be done, I am getting a 'Tried to synchronously call a non-worklet function on the UI thread'